package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.8",
+  "version": "0.5.9",
```

Comment:
Sorry, now we already have a .8 and .9. Please now update to .10.

Reply:
👍

Reply:
Or check with @jachurchill to see if he will take 10 (and you would take 11)... or just update this last before your PR is merged.

Reply:
👍